Build failure, undefined reference to cblas_*

I'm trying to build kaldi on Alpine Linux which uses Musl libc rather than glibc. I managed to make it compile with `configure` by setting `LDFLAGS="$LDFLAGS -lcblas -llapack"` but the same does not seem to work with CMake. Of course, blas and lapack are installed.


Why does it fail to link properly to blas?

```
ninja: job failed: : && /usr/lib/ccache/bin/g++ -Os -fomit-frame-pointer -Os -DNDEBUG -Wl,--as-needed -lcblas -llapack src/matrix/CMakeFiles/sparse-matrix-test.dir/sparse-matrix-test.cc.o -o src/matrix/sparse-matrix-test  -Wl,-rpath,/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/matrix:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/base:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/_deps/openfst-build/src/lib:  -ldl  src/matrix/libkaldi-matrix.so  src/base/libkaldi-base.so  -ldl  /usr/lib/libblas.so  /usr/lib/liblapack.so  _deps/openfst-build/src/lib/libfst.so.16 && :
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_stpsv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgemm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspr2'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgemv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ddot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sdot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sscal'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dsyrk'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspr2'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_scopy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspr'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgbmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ssymm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_saxpy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dger'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspr'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ssyrk'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgemv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgbmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dcopy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_drot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dtpsv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sger'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_srot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dsymm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_stpmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dtpmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dscal'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgemm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_daxpy'
collect2: error: ld returned 1 exit status
ninja: job failed: : && /usr/lib/ccache/bin/g++ -Os -fomit-frame-pointer -Os -DNDEBUG -Wl,--as-needed -lcblas -llapack src/matrix/CMakeFiles/numpy-array-test.dir/numpy-array-test.cc.o -o src/matrix/numpy-array-test  -Wl,-rpath,/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/matrix:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/base:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/_deps/openfst-build/src/lib:  -ldl  src/matrix/libkaldi-matrix.so  src/base/libkaldi-base.so  -ldl  /usr/lib/libblas.so  /usr/lib/liblapack.so  _deps/openfst-build/src/lib/libfst.so.16 && :
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_stpsv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgemm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspr2'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgemv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ddot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sdot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sscal'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dsyrk'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspr2'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_scopy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspr'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgbmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ssymm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_saxpy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dger'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspr'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ssyrk'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgemv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgbmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dcopy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_drot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dtpsv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sger'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_srot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dsymm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_stpmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dtpmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dscal'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgemm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_daxpy'
collect2: error: ld returned 1 exit status
ninja: job failed: : && /usr/lib/ccache/bin/g++ -Os -fomit-frame-pointer -Os -DNDEBUG -Wl,--as-needed -lcblas -llapack src/util/CMakeFiles/text-utils-test.dir/text-utils-test.cc.o -o src/util/text-utils-test  -Wl,-rpath,/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/util:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/matrix:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/src/base:/home/bart/Documents/Git/alpine/aports/testing/kaldi/src/kaldi-f6f4ccaf213f0fe8b26e633a7dc0c802150626a0/build/_deps/openfst-build/src/lib:  -ldl  src/util/libkaldi-util.so  src/matrix/libkaldi-matrix.so  src/base/libkaldi-base.so  -ldl  /usr/lib/libblas.so  /usr/lib/liblapack.so  _deps/openfst-build/src/lib/libfst.so.16 && :
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_stpsv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgemm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspr2'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgemv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ddot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sdot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sscal'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dsyrk'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspr2'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_scopy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspr'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sgbmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ssymm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_saxpy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dspmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dger'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspr'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_ssyrk'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgemv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgbmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dcopy'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_drot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dtpsv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sger'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_srot'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dsymm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_stpmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dtpmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_sspmv'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dscal'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_dgemm'
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: src/matrix/libkaldi-matrix.so: undefined reference to `cblas_daxpy'
```

--- a/cmake/gen_cmake_skeleton.py
+++ b/cmake/gen_cmake_skeleton.py
@@ -269,7 +269,7 @@ class CMakeListsLibrary(object):
 
         if len(self.depends) > 0:
             ret.append("target_link_libraries(" + self.target_name + " PUBLIC")
-            for d in self.depends:
+            for d in self.depends + ['-lcblas', '-llapack']:
                 ret.append("    " + d)
             ret.append(")\n")
 

